doc/man3/CRYPTO_get_ex_new_index.pod

```diff
@@ -52,8 +52,10 @@ The specific structures are:
     RSA
     ENGINE
     UI
-    UI_METHOD
```

Comment:
I'd rather that stayed there, since `UI` and `UI_METHOD` belong together.

Reply:
You are right: It does not make sense to order the names according to the numerical order of the `CRYPTO_EX_INDEX_*` constants in `<openssl/crypto.h>`. I added a fixup that reorders the names alphabetically (f8a576fef0fd).